taker/src/routes.rs

```diff
@@ -57,37 +56,9 @@ pub struct IdentityInfo {
     pub(crate) taker_peer_id: String,
 }
 
-#[derive(Debug, Copy, Clone, strum_macros::Display)]
-pub enum ContractSymbol {
```

Comment:
🙃 I think having a dedicated struct for the API here is actually a good thing, because we can decide the serialization can be guaranteed in a better way. (e.g. not breaking the API with renaming a field)

Reply:
I agree on principal, but this was removed because it (or anything like it) is not used at all after these changes.

Reply:
My concern was that we expose `model::ContractSymbol` as part of `projection::CfdOffers`, but that was already the case before.
Please ignore this for now.